actors/runtime/src/runtime/policy.rs

```diff
@@ -0,0 +1,316 @@
+use std::collections::HashSet;
+
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::sector::{RegisteredPoStProof, RegisteredSealProof};
+
+// A trait for runtime policy configuration
+pub trait RuntimePolicy {
+    fn policy(&self) -> &Policy;
+}
+
+// The policy itself
+pub struct Policy {
```

Comment:
Is it possible to replicate the all caps filed names, i.e. `MAX_AGGREGATED_SECTORS`?  Even with the `policy.` prefix it would be nice to keep this immediate visual cue that we are looking at protocol constants.

Reply:
rust doesnt like that one bit.

We can probably mute the compiler and do it, but i dont think it would be considered idiomatic rust any more.